CHANGELOG.md

```diff
@@ -66,6 +66,7 @@ Here is an overview of all new **experimental** features:
 - **Hashicorp Vault**: Add support to get secret that needs write operation (e.g. pki) ([#5067](https://github.com/kedacore/keda/issues/5067))
 - **Kafka Scaler**: Ability to set upper bound to the number of partitions with lag ([#3997](https://github.com/kedacore/keda/issues/3997))
 - **Kafka Scaler**: Add support for Kerberos authentication (SASL / GSSAPI) ([#4836](https://github.com/kedacore/keda/issues/4836))
+- **Openstack Scaler**: Use Gophercloud SDK ([#4962](https://github.com/kedacore/keda/issues/3439))
```

Comment:
Please move this to Other section, as it is not user facing change

Reply:
Get this error now:
`Checking section: Unreleased
Error: Wrong pattern found in section: Other , line: - **Openstack Scaler**: Use Gophercloud SDK ([#4962](https://github.com/kedacore/keda/issues/3439))`